client/src/index.js

```diff
@@ -1,11 +1,14 @@
+import "materialize-css/dist/css/materialize.min.css";
```

Comment:
I don't think you included this file in git so its not on github for me to download and its causing the app not to run.
just because it is on your computer doesn't mean that it goes into source control so that other people can use it.
this happens all the time

Reply:
Can you please take another look at it? I just moved the materialize file from the node_modules to my public css so it be read.  Not sure why it doesn't show up when you yarn start.  Please I worked too hard just to get a zero on it.